easybuild/tools/config.py

```diff
@@ -147,6 +147,7 @@ def mk_full_default_path(name, prefix=DEFAULT_PREFIX):
         'skip_test_cases',
         'sticky_bit',
         'upload_test_report',
+        'update',
```

Comment:
We need a better name than `--update`, since it's too confusing w.r.t. *what* is being updated; it's not clear that this is about updating EasyBuild itself (or actually, installing a module for the latest EasyBuild version, it's not even in-place updating).